.kitchen.yml

```diff
@@ -17,6 +17,7 @@ suites:
   - name: default
     data_bags_path: test/integration/default/data_bags
     run_list:
+      - recipe[apt]
```

Comment:
Do we need `recipe[apt]` in this one?

Reply:
My tests were failing without it because it couldn't find the packages at their latest versions and this resolved that problem. I'm totally open to a different solution if you feel there's a better approach.

Reply:
Alright! I'll just go ahead and merge it.

It's been a while, and I also don't have a working environment so I'm trusting you've tried this out for real!

Reply:
haha, I promise I did! I'd be happy to provide screenshots if you'd like, I completely understand the desire to be thorough